### ISSUE TITLE
MinNary issue after upgrading tract

### ISSUE THREAD

**Issue by hrydgard** [sonos/tract#420]
I am trying to upgrade our project from using tract 0.6.3 to tract 0.11.2. Impressively, to get it to compile again, the only change I had to do to our code was:

```
-    Some(shape.dim(1)?.concretize()?.as_const()? as usize)
+    Some(shape.dim(1)?.concretize()?.to_i64().ok()? as usize)
```

but hitting the following snag trying to load our ONNX files:

```Translating node #21 "Min__31206" MinNary ToTypedTranslator -> Translating node #21 "Min__31206" MinNary ToTypedTranslator -> wiring Min__31206-0 (TypedBinOp(Min)) -> Typed ops require rank match. Invalid inputs for Min: [1x22xF32, xF32 2]```

I'm not really sure what to do here - did the MinNary node change in some incompatible way? Something wrong with our files that accidentally worked before?

**Comment by kali** [comment]
This is probably on my side. tract-core became more strict about rank broadcasting over the past months... I can have a look and probably make a blind fix, but it would help if you can provide me a model, even a non-trained one of course.

**Comment by hrydgard** [comment]
Cool. Do you have a way I can send you the .onnx privately?

**Comment by kali** [comment]
You can mail it to mathieu (at) poumeyrol (dot) fr if it is small enough. Otherwise, we'll find something else.

**Comment by hrydgard** [comment]
This one is 400k-ish, should be fine :)

**Comment by kali** [comment]
Got it. thanks.

**Comment by kali** [comment]
FYI, fix released with 0.12.1

**Comment by hrydgard** [comment]
Just confirming that the fix works for us as expected. Thanks!
